api_app/analyzers_manager/file_analyzers/guarddog_file.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from api_app.analyzers_manager.classes import DockerBasedAnalyzer, FileAnalyzer
+from tests.mock_utils import MockUpResponse
+
+logger = logging.getLogger(__name__)
+
+
+class GuarddogFile(FileAnalyzer, DockerBasedAnalyzer):
+    name: str = "Guarddog"
+    url: str = "http://malware_tools_analyzers:4002/guarddog"
+    # http request polling max number of tries
+    max_tries: int = 15
+    # interval between http request polling (in secs)
+    poll_distance: int = 30
```

Comment:
This is the polling between this script in the main docker container and the malware docker container. The problem about this number is that if the analyzers finishes its computation in the middle of those 30 seconds, the client would get the results a lot later than intended. Because of this, this value can usually reduced to a lower number.
How much time does this analyzer require to be completely executed in your experience?